.pylintrc

```diff
@@ -41,10 +41,11 @@ good-names=i,
 # Number of spaces of indent required inside a hanging or continued line.
 indent-after-paren=2
 
-# String used as indentation unit. This is usually "    " (4 spaces) or "\t" (1
-# tab).
+# String used as indentation unit. This is usually "    " (4 spaces) or "\t" (1 tab).
 indent-string='  '
 
+max-line-length=120
```

Comment:
:heart:

Reply:
You're welcome haha. Note that autopep8 was set to a line length of 120 recently, but pylint's setting got missed.